### ISSUE TITLE
Set_tasks_magnitude_and_phase modification

### ISSUE THREAD

**Issue by NLopezRios** [shimming-toolbox/CST-simulation-macros#2]
Would it be possible to modify the Set_tasks_magnitude_and_phase macro so that the phases of the Extra Tasks ("CPmode", "negCPMode" and "zeroPhase") can be adjusted by entering the project-specific CP phases extracted from the experimental data, or at least warn/prompt the user to do so?

**Comment by kylemgilbert** [comment]
In `Set_tasks_magnitude_and_phase`, the variable `Phases` refers to the list of phase offsets calculated through the VOP code for every channel. The `CPmodePhases` refers to the experimental CP mode phases (i.e., what you would set on the scanner for TrueForm mode. Is this already what you want? Perhaps we just need better commenting in the code?

**Comment by NLopezRios** [comment]
Yes, but it looks like I didn't use the most recent versions, because I don't see CPmodePhases in them. Good to know it's done. I downloaded them on Nov 15, when I started working on the new VOPs and I heard that you had already modified them.
